experimental/packages/opentelemetry-sdk-node/package.json

```diff
@@ -54,10 +54,10 @@
     "@opentelemetry/sdk-trace-node": "~1.1.1"
```

Comment:
Is it intended to use ~ here even for components which are now in the same lerna project?

Reply:
I think we should probably pin them